src/runtime/web/index.js

```diff
@@ -1,14 +1,8 @@
 'use strict';
 
-import io from './io';
-import Env from '../../lib/env';
-import { L10n } from '../../bindings/html/service';
-import { View } from '../../bindings/html/view';
-import { getMeta } from '../../bindings/html/head';
-import {
-  changeLanguage, onlanguagechage, onadditionallanguageschange,
-  getAdditionalLanguages
-} from '../../bindings/html/langs';
+import { fetch } from './io';
+import { L10n, init } from '../../bindings/html/service';
+import { getAdditionalLanguages } from '../../bindings/html/langs';
```

Comment:
why are you loading getAdditionalLanguages here, just to pass it to service init?